src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -11,7 +11,7 @@
 public class CommandResult {
 
     /** The feedback message to be shown to the user. Contains a description of the execution result */
-    public final String feedbackToUser;
+    private final String feedbackToUser;
```

Comment:
Hi Boon Joon, I do believe that this is a task in Activity T3A2 Encapsulation, do take note to only push related materials with accordance to the tasked activity in the future :) No worries for this time round!